NAMESPACE

```diff
@@ -124,6 +124,10 @@ export(expect_type)
 export(expect_vector)
 export(expect_visible)
 export(expect_warning)
+export(expect_without_condition)
```

Comment:
I think these names are a bit too long. They will dominate the line width, making it harder to read the actual logic.

Reply:
Might be too literary but:

`expect_sans_warnings(foo(), class = "foo")`?

Reply:
Or too French 😛

Reply:
> `expect_no_warnings(foo(), class = "abc")` doesn't make as much sense to me.

I don't love the "no" particle but I like the concision.
Does it make sense to read it as "expect no warnings of class 'abc'"?

Reply:
Ok, you've convinced me